how-to-docs/examples/rds-temp.yaml

```diff
@@ -52,10 +52,12 @@ Resources:
         FromPort: 5432
         ToPort: 5432
         SourceSecurityGroupId: {{ get_stack_output(cf_conn, env + '-infra', 'ComputeDefaultSG') }}
+{% if vpn_access %}
       - IpProtocol: tcp
```

Comment:
you should have vpn_acccess set to false by default top with some kind of explanation